app/controllers/comments_controller.rb

```diff
@@ -50,20 +51,17 @@ def update
 
   # DELETE /comments/1 or /comments/1.json
   def destroy
-    @comment.destroy
```

Comment:
I think you still need that though. What happens when I try to delete a comment?